src/lib/kclCommands.ts

```diff
@@ -178,6 +180,11 @@ export function kclCommands(commandProps: KclCommandConfig): Command[] {
       groupId: 'code',
       hide: 'web',
       needsReview: true,
+      reviewValidation: async () => {
+        if (commandProps.kclManager.isExecuting) {
+          return new Error(EXECUTING_MESSAGE)
+        }
+      },
```

Comment:
I'm not too sold on this fix because I feel like we should check on this more generally? The good thing here is that no matter how this command gets triggered we check before submitting 🤷

Reply:
I'm fine with this fix but I believe it could mean long wait times for users with large assemblies trying to add items, correct? They'd have to wait for execution to complete before inserting?

Reply:
>We don't really have that problem elsewhere because most codemods are modeling commands, which you can't enter during execution. But Insert is part of kclCommands, which doesn't have that guard

Ohhh thanks I did not realize this.